pkg/operator/ceph/csi/provisioner/controller.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provisioner
+
+import (
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/coreos/pkg/capnslog"
+
+	"google.golang.org/grpc"
+
+	"github.com/rook/rook/pkg/clusterd"
+
+	ctrl "github.com/kubernetes-csi/external-provisioner/pkg/controller"
+	snapclientset "github.com/kubernetes-csi/external-snapshotter/pkg/client/clientset/versioned"
+	"github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/controller"
+	csiclientset "k8s.io/csi-api/pkg/client/clientset/versioned"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	csiEndpoint            = "/var/lib/kubelet/plugins/csi-rbdplugin/csi-provisioner.sock"
+	connectionTimeout      = 10 * time.Second
+	volumeNamePrefix       = "pvc"
```

Comment:
in order to distinguish from PVs provisioned by ceph-csi, we can use `rook-csi-pvc` prefix here

Reply:
is there a pattern for other provisioners adding a named prefix? Perhaps `rook-pvc` would be good. Long term, everything will move to csi so we shouldn't need `csi` in the name, right?